### ISSUE TITLE
Missing class properties transform.

### ISSUE THREAD

**Issue by madamadore** [martijnversluis/ChordSheetJS#662]
Hi, I have this error while I try to import this library. Maybe I misconfigured my environment. Can someone help, please?

Many thanks
M

node_modules/chordsheetjs/lib/module.js: Missing class properties transform.
  1958 | ];
  1959 | class $918eb2016ac61e41$var$Key {
> 1960 |     modifier = null;
       |     ^
  1961 |     minor = false;
  1962 |     static parse(keyString) {
  1963 |         return (0, $974bd1d260bb1d53$export$595e981f61839426)(keyString, $918eb2016ac61e41$var$Key, $918eb2016ac61e41$var$regexes);

**Comment by martijnversluis** [comment]
Hey @madamadore,

Thanks for reaching out. 

I'd like to try to reproduce your issue to find the cause. Could you elaborate a bit on what your setup looks like? Or link to a repo I can check out?

**Comment by martijnversluis** [comment]
Hey @madamadore. Is this still an issue? If so, could you provide some info concerning your setup?

**Comment by madamadore** [comment]
Hi @martijnversluis !
Thanks for your reply. Unfortunately, I am not able to reproduce that error. Since August I made other steps and I temporarily solved my problems avoiding to use your library. Now I am trying to integrate it again but I am fighting with Babel and WebPack! 😆 
I think that my problems were caused by these tools indeed, which were not properly configurated. I'll let you know!
Many thanks! 🙏 
M
